web/src/actions/user.js

```diff
@@ -37,11 +37,9 @@ export const updateUser = (id, data) => dispatch => {
   return axios
     .put(`${API_URL}/users/${id}`, data)
     .then(req => {
-      console.log('update user success!', req);
```

Comment:
thx

Reply:
I was just getting tired of the console output when I was running the tests.  😆